Sources/WebDriver/Session.swift

```diff
@@ -317,6 +317,19 @@ public class Session {
         try webDriver.send(Requests.SessionWindowSize.Post(session: id, windowHandle: handle, width: width, height: height))
     }
 
+    /// - Parameters:
+    ///   - latitude: Number coordinate of current geo location
+    ///   - longitude: Number coordinate of current geo location
+    ///   - altitude: Number coordinate of current geo location
+    public func setLocation(latitude: Int, longitude: Int, altitude: Int) throws {
+        try webDriver.send(Requests.SessionGeoLocation.Post(session: id, latitude: latitude, longitude: longitude, altitude: altitude))
+    }
+
+    public func getLocation() throws -> (latitude: Int, longitude: Int, altitude: Int) {
```

Comment:
It's worth defining a Location struct and taking it as an argument in the setter. If you make it Codable, you can use that same struct as the Body of the post and the ResponseValue of the get requests.